.github/workflows/codacy.yml

```
@@ -0,0 +1,82 @@
+          # You can also omit the token and run the tools that support default configurations
+          project-token: ${{ secrets.CODACY_PROJECT_TOKEN }}
+          verbose: true
+          output: results.sarif
+          format: sarif
+          # Adjust severity of non-security issues
+          gh-code-scanning-compat: true
+          # Force failure on any issues found
+          max-allowed-issues: 0
+
+      - name: Fail if Codacy found issues
+        if: (steps.parse_sarif.outputs.has_major == 'true' || steps.parse_sarif.outputs.has_critical == 'true')
```

actionlint (1.7.7)

property "parse_sarif" is not defined in object type {codacy: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)